kevm-pyk/src/kevm_pyk/kproj/evm-semantics/lemmas/lemmas.k

```diff
@@ -22,15 +26,16 @@ module LEMMAS [symbolic]
     rule 0 <=Int #ceil32(I)        => true requires 0 <=Int I [simplification, smt-lemma]
     rule 0 <=Int #ceil32(I) -Int I => true requires 0 <=Int I [simplification]
 
-    rule X *Int Y <Int pow256 => true requires Y <=Int maxUInt256 /Int X [simplification]
+    rule X *Int Y <Int pow256 => true requires Y <=Int maxUInt256 /Int X
+      requires 0 <=Int X orBool 0 <=Int Y [simplification]
```

Comment:
Is this a typo? It looks like we have `requires ...` twice....

Reply:
Yes, that's quite horrific, I didn't notice.

Reply:
Fixed.